.github/workflows/refresh-base-image.yaml

```
@@ -0,0 +1,82 @@
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  Refresh_Base_Image:
+    runs-on: [mend-self-hosted, profile=developer-platform-xlarge-od]
+
+    steps:
+      - name: Checkout Release Branch
+        uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)